src/Draco.Compiler/Internal/Symbols/VariableSymbol.cs

```diff
@@ -14,4 +16,20 @@ internal abstract partial class VariableSymbol : Symbol, ITypedSymbol
     /// True, if this local is mutable.
     /// </summary>
     public abstract bool IsMutable { get; }
+
+    public override Api.Semantics.Visibility Visibility
```

Comment:
Do we want to move this into GlobalSymbol instead? Is there any other variable where visibility is significant?
It doesn't matter that much, but for the other symbols where this doesn't matter this is a slight slowdown.